src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -378,7 +378,7 @@ class LazyClassLoaderHelper
                 // set, method lookup succeeds.
 
                 // Otherwise, if the maximally-specific superinterface methods (§5.4.3.3) of C for the name and
-                // descriptor specified by the method reference include exactly one method that does not have its
```

Comment:
This is just a verbatim copy from the spec and IIRC correct english without the `to`

Reply:
Sorry, autocorrect